### ISSUE TITLE
Could not find module ember-metal/streams/stream in ember-2.7.0-beta.1

### ISSUE THREAD

**Issue by mike-north** [ember-animation/liquid-fire#464]
When using the latest release of liquid fire (0.23.1) with the newly-released ember `2.7.0-beta.1`, the user gets an error 
```
Uncaught Error: Could not find module ember-metal/streams/stream
```

### Steps to reproduce

```sh
ember new liquid-27 && \
 cd liquid-27 && \
 rm -rf bower_components/ember && \
 bower install --save ember#beta && \
 ember install liquid-fire && \
 ember g route home && \
 ember g acceptance-test home && \
 ember test ci
```

You should see your new acceptance test fail, due to the aforementioned missingModule error

**Comment by mike-north** [comment]
Just tested w/ `63481e6415d496a43741d458daf7e90c4edd084a` (master as of now) and get the same error.

**Comment by ntodd** [comment]
0.24.1 appears to resolve this issue.

**Comment by msalahz** [comment]
Try re-initing and upgrading all of your dependencies as mentioned here https://github.com/ember-cli/ember-cli/issues/6063

**Comment by mike-north** [comment]
Closing this, since it is resolved in the `0.24.x` release series
